ionoscloud/resource_server.go

```diff
@@ -1373,7 +1384,12 @@ func getServerData(d *schema.ResourceData) (*ionoscloud.Server, error) {
 			server.Properties.CpuFamily = &vStr
 		}
 	}
-
+	if v, ok := d.GetOk("hostname"); ok {
+		if v.(string) != "" {
```

Comment:
@cristiGuranIonos I see that we have empty string checks for some fields here.

With the current code (with validations) this will never be the case, the string will never be empty.

If you choose the remove the validation as I suggested in a comment above and also remove this empty string check, the request will be populated with an empty string `hostname` and the API request will fail with a validation error, there is nothing wrong with that.

With or without validation, I suggest removing this check, not sure how it helps.